src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -188,6 +188,7 @@ private void focusTextField(Event event) {
     private void setEntryTypeForReturnAndClose(Optional<BibEntryType> entryType) {
         type = entryType.map(BibEntryType::getType).orElse(null);
         viewModel.stopFetching();
+        this.basePanel.frame().getGlobalSearchBar().endSearch();
```

Comment:
A better solution would be to Inject the StateManager and call 
`clearSearchQuery`